### ISSUE TITLE
Ability to empty Trash automatically.

### ISSUE THREAD

**Issue by gif-byte** [standardnotes/forum#1094]
I would like to have the ability to automatically delete notes that have been in the trash for a month.

**Comment by mobitar** [comment]
I'm torn about this feature. On the one hand, I can see how this can be useful. On the other hand, I really don't like the idea of a script automatically and permanently deleting your notes. I feel like this is something you should do on your own. Similar to how OS's don't empty your trash bin until you explicitly empty them.

**Comment by joshcsullivan** [comment]
I agree with @mobitar on this request. I definitely do not want something automatically deleting notes from my account, even if they are already in the trash.
